client.yml

```diff
@@ -1550,6 +1546,10 @@ paths:
       format: uuid
       name: token
       in: query
+    - type: string
+      format: enrollment-method
```

Comment:
If I understand correctly, the optional `format` modifier for [data types](https://swagger.io/specification/v2/#data-types) is an open string, so I think it can have any value that we deem useful for documentation, `uuid` is an example of a format that's not defined in OAS2. Still, I've removed that `format` modifier because it doesn't seem to be adding much value as an arbitrary string. It's pretty clear from the query param name "method" and the valid values what it's for.